### PR TITLE
updating the normalize

### PR DESCRIPTION
**Pull request by aashokvardhan**

### CODE REVIEW

File: apphub/instance_segmentation/solov2/solov2_torch.py [hunk 1]
```diff
@@ -585,8 +585,8 @@ def get_estimator(data_dir=None,
     init_lr = 1e-2 / 16 * batch_size
     model = fe.build(model_fn=SoloV2, optimizer_fn=lambda x: torch.optim.SGD(x, lr=init_lr, momentum=0.9))
     network = fe.Network(ops=[
-        Normalize(inputs="image", outputs="image", mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
         Permute(inputs="image", outputs='image'),
```
Comment (vbvg2008) [inline apphub/instance_segmentation/solov2/solov2_torch.py:588]: Now that permute is no longer necessary, we should remove the permute and use ChannelTranspose in Pipeline.

File: fastestimator/backend/_tensor_normalize.py [hunk 2]
```diff
@@ -67,11 +68,15 @@ def normalize(tensor: Tensor,
...
-    mean = get_scaled_data(mean, max_pixel_value, framework, device)
-    std = get_scaled_data(std, max_pixel_value, framework, device)
+    mean = convert_tensor_precision(get_scaled_data(mean, max_pixel_value, framework, device))
+    std = convert_tensor_precision(get_scaled_data(std, max_pixel_value, framework, device))
+    tensor = convert_tensor_precision(tensor)
 
-    tensor = (convert_tensor_precision(tensor) - convert_tensor_precision(mean)) / (convert_tensor_precision(std) +
-                                                                                    epsilon)
+    if framework == 'torch':
+        transform = T.Normalize(mean=mean, std=std + epsilon)
```
Comment (vbvg2008) [inline fastestimator/backend/_tensor_normalize.py:76]: tensor = T.Normalize(mean=mean, std=std + epsilon)(tensor)

**Review by TortoiseHam — APPROVED** [review]
LG
